packages/web/src/pages/track-page/components/mobile/TrackHeader.tsx

```diff
@@ -486,8 +503,15 @@ const TrackHeader = ({
       <div className={cn(styles.infoSection, styles.withSectionDivider)}>
         {renderTrackLabels()}
       </div>
-      {renderDownloadButtons()}
       {renderTags()}
+      {!isLosslessDownloadsEnabled ? renderDownloadButtons() : null}
```

Comment:
hang on, wouldn't this render the dl buttons beneath the tags? isn't that opposite of what it was?

Reply:
yup, that's right :) in the mocks!